src/hooks/index.js

```diff
@@ -1,11 +1,12 @@
 import useField from './useField';
```

Comment:
I'd prefer the imports and exports be alphabetical, so it's easier to find what you need to adjust/delete/replace/etc - and so it's consistent with all the other files.  Even though, it doesn't matter one bit :)

Reply:
fixed